src/Compilers/CSharp/Portable/Symbols/TypeWithAnnotations.cs

```diff
@@ -1022,4 +1026,83 @@ internal override bool TypeSymbolEquals(TypeWithAnnotations type, TypeWithAnnota
             }
         }
     }
+
+    /// <summary>
+    /// Represents a sequence of <see cref="NullableAnnotation"/> to apply to a type. The sequence of
+    /// flags is specific to how the individual <see cref="TypeSymbol"/> walk their nested types.
+    /// </summary>
+    internal sealed class NullableTransformStream
+    {
+        private static readonly ObjectPool<NullableTransformStream> Pool = new ObjectPool<NullableTransformStream>(() => new NullableTransformStream(), Environment.ProcessorCount);
+
+        private ImmutableArray<byte> _transforms;
```

Comment:
>private [](start = 8, length = 7)

readonly

Reply:
This can be mutated as it's a pooled object.


---
In reply to: [274613981](https://github.com/dotnet/roslyn/pull/34891#discussion_r274613981) [](ancestors = 274613981)